### ISSUE TITLE
Making changes to files in the wizard

### ISSUE THREAD

**Issue by tshort** [JuliaPackaging/BinaryBuilder.jl#107]
This is probably a documentation issue. In the wizard, I struggled with how to make changes to files. If you just `vi` something, it doesn't leave any record in the build script.

For a file `file.ext`, my approach was something like this:

```
cp file.ext file.ext.orig
vi file.ext     # make the changes
diff -u file.ext.orig file.ext
# Create a patch based on the results copy-pasted from the output of `diff`
cat > file.patch <<'END'
--- file.ext.orig 2017-12-14 19:28:48.816021000 -0500
+++ file.ext2017-12-14 19:29:03.912021000 -0500
@@ -1,4 +1,5 @@
 -https://computation.llnl.gov/projects/sundials/download/sundials-3.0.0.tar.gz
 +https://computation.llnl.gov/projects/sundials/download/sundials-3.1.0.tar.gz
  
  http://faculty.cse.tamu.edu/davis/SuiteSparse/SuiteSparse-5.0.0.tar.gz
 
END
# Apply the patch
patch -l file.ext.orig file.patch -o file.ext
```
Any better ways to approach this?

**Comment by tshort** [comment]
See #125.

**Comment by aviks** [comment]
I've been writing sed scripts by hand, but yeah, it takes a couple of iterations to get right.

**Comment by felixcremer** [comment]
Is this closed by https://github.com/JuliaPackaging/BinaryBuilder.jl/pull/678?

**Comment by giordano** [comment]
> Is this closed by #678?

I'd say yes, thanks for pointing this out!
